oauth2_http/javatests/com/google/auth/oauth2/DefaultCredentialsProviderTest.java

```diff
@@ -176,6 +166,63 @@ public void getDefaultCredentials_noCredentials_singleGceTestRequest() {
         ComputeEngineCredentials.MAX_COMPUTE_PING_TRIES);
   }
 
+  @Test
+  public void getDefaultCredentials_noCredentials_linuxNotGce() throws IOException {
+    TestDefaultCredentialsProvider testProvider = new TestDefaultCredentialsProvider();
+    testProvider.setProperty("os.name", "Linux");
+    String productFilePath = "/sys/class/dmi/id/product_name";
```

Comment:
Move this to constants. Maybe good to define a constants file instead of dumping in OAuth2Utils

Reply:
Will move to final. 

We do not dump constant to OAuthUtils, unless those contants are util related, like transport. Usually constants sits where they belong. Like just the DefaultCredentialsProvider has 16 constants vs 12 in OAuthUtils.